utcaApp/src/Makefile

```diff
@@ -34,11 +34,9 @@ utca_DBD += aSubRoutines.dbd
 
 utca_DBD += menuScan.dbd
 
-utca_LIBS += uhal-modules uhal-utilities
-uhal-utilities_DIR = $(UHAL_LIBS)
-uhal-modules_DIR = $(UHAL_LIBS)
+utca_SYS_LIBS += uhal-modules uhal-utilities
 
-OP_SYS_INCLUDES += -I$(UHAL)/include/uhal
+OP_SYS_INCLUDES += -I/usr/local/include/uhal
```

Comment:
We can't rely on the preprocessor finding it by its own due to lack of the `uhal` namespace in all includes, right? A cleaner approach would be to do so (just like any other system library). This would remove the requirement for it to be installed at this specific system path and so on.

Reply:
I'm not sure I understand. What are you suggesting? Doing `#include <uhal/...>` in source files?